storage/schemadb/tests/db.rs

```diff
@@ -20,7 +20,7 @@ define_schema!(TestSchema1, TestField, TestField, "TestCF1");
 define_schema!(TestSchema2, TestField, TestField, "TestCF2");
 
 #[derive(Debug, Eq, PartialEq)]
-struct TestField(u32);
+pub struct TestField(u32);
```

Comment:
they need to be accessed outside of its own crate, cuz we expose db initialization in a different service, instead of the db storage crate. but they will be eventually reverted back after we deleting the old indexer db.

Reply:
I mean, where? Why do you need TestField outside of this crate?

Reply:
we don't use `TestField` directly, but i think `TestField` requires `define_schema` and we use `define_schema` in a different cargo.